flink-external-resource/flink-external-resource-gpu/src/test/resources/test-gpu-discovery.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+if [ $# -lt 1 ]; then
+  echo "Usage: ./gpu-discovery-script.sh gpu-amount [exit-non-zero]"
+  exit 1
+fi
+
+AMOUNT=$1
+
+if [ $AMOUNT -eq 0 ]; then
+  exit 0
+fi
+
+if [ "$2" == "exit-non-zero" ]; then
+  exit 1
+fi
+
+
+normal_allocate() {
+  indexes=$1
+  occupy_indexes=()
+  for i in $indexes
+    do
+      if [ ${#occupy_indexes[@]} -lt $AMOUNT ]; then
+        occupy_indexes[${#occupy_indexes[@]}]=$i
+      fi
+    done
+    if [ $AMOUNT -gt ${#occupy_indexes[@]} ]; then
+      echo "Could not get enough GPU resources."
+      exit 1
+    fi
+    echo ${occupy_indexes[@]} | sed 's/ /,/g'
+}
+
+indexes=($(seq 0 1 $AMOUNT))
+normal_allocate "${indexes[*]}"
```

Comment:
Can this script simply always return `seq ${AMOUNT} | paste -sd ',' -`, unless `exit-non-zero`?